package.json

```diff
@@ -52,7 +52,7 @@
     "test:unit": "jest test/unit",
     "validate": "componentsjs-compile-config urn:solid-server:default -c config/config-default.json -f > /dev/null",
     "version": "manual-git-changelog onversion",
-    "watch": "nodemon --watch \"src/**/*.js\" --watch \"bin/**/*.js\" --exec npm start"
```

Comment:
Let's remove the `nodemon` dependency as well.

Reply:
👍